2.4.2 Page Titled - Not all non-web docs and SW have titles that describe the topic or purpose

maryjom opened the pull request:
This is a reimplementation of PR #633 using the current main branch as the original source which was approved by the TF back in May but not incorporated because we hadn't solved non-web documents yet. This new PR is to address issue #627 - for both non-web documents and software since there are problems applying this SC in both cases.

maryjom committed:
Update comments-by-guideline-and-success-criterion.md

This is a reimplementation of PR #633 using the current main branch as the original source which was approved by the TF back in May but not incorporated because we hadn't solved non-web documents yet. This new PR is to address issue #627 - for both non-web documents and software since there are problems applying this SC in both cases.

maryjom committed:
New guidance for software eliminates need for bullet in problematic for closed

daniel-montalvo reviewed (changes requested):
see my suggestions
above

maryjom committed:
Proposal 3: A variant using Gregg and Mike's input

Gregg, Mike, and I had an email thread to develop potential language. The alternate requirement isn't exactly as Gregg had it but the note is what was proposed. I'm trying to keep this more closely aligned to WCAG2ICT's writing style which doesn't use the precondition style used in the EN 301 549.

maryjom committed:
Pulling in Daniel's editorial change as well

maryjom committed:
Update PR per the 24 October meeting

maryjom committed:
Change "allows" to "specifies" per Daniel's comment

This is an editorial change that should not affect the CfC.

maryjom committed:
Merge branch 'main' into mjm-page-titled-guidance

maryjom committed:
per resolution on 30 Oct, remove phrase

Co-authored-by: Daniel Montalvo <49305434+daniel-montalvo@users.noreply.github.com>

maryjom committed:
Merge branch 'main' into mjm-page-titled-guidance

maryjom committed:
Merge branch 'main' into mjm-page-titled-guidance

maryjom committed:
Merge branch 'main' into mjm-page-titled-guidance

daniel-montalvo reviewed (approved):
Approving, just a minor editorial tweak for software on a suggestion above.

maryjom commented on comments-by-guideline-and-success-criterion.md:
```diff
@@ -604,24 +604,20 @@ See also the [Comments on Closed Functionality](#comments-on-closed-functionalit
-<div class="note wcag2ict software">
-    
-See also the [Comments on Closed Functionality](#comments-on-closed-functionality).</div>
+This success criterion is problematic to apply directly to non-web software through simple word substitution because application titles rarely describe the topic or purpose of the software. However, where the platform supports a programmatic title or name for a software window or screen, when a software application utilizes that feature to provide a unique title or name for each window or screen, ...
```
@daniel-montalvo Non-web documents have the "and/or" on a different phrase that is not in the non-web software explanation. For non-web documents it is used on the phrase "and/or when a meaningful file name can be supplied" which is an alternate to utilizing the document technology's capability to supply a title or name inside of the document. So I don't think this change to use "and/or" should be made for non-web software.
